src/interfaces/wallet.cpp

```diff
@@ -99,8 +99,6 @@ WalletTx MakeWalletTx(interfaces::Chain::Lock& locked_chain, CWallet& wallet, co
 //! Construct wallet tx status struct.
 WalletTxStatus MakeWalletTxStatus(interfaces::Chain::Lock& locked_chain, const CWalletTx& wtx)
 {
-    LockAnnotation lock(::cs_main); // Temporary, for CheckFinalTx below. Removed in upcoming commit.
```

Comment:
This is also used for `::mapBlockIndex`, so can not be removed yet?

Reply:
re: https://github.com/bitcoin/bitcoin/pull/15288#discussion_r255300183

> This is also used for `::mapBlockIndex`, so can not be removed yet?

Seems like a silent merge conflict with #15321 (this PR has not been rebased since that was merged).